README.md

```diff
@@ -39,6 +39,9 @@ By default this package will enable @iopipe/trace and @iopipe/event-info plugins
 - [IOpipe Documentation](https://github.com/iopipe/iopipe-js-core#readme)
 - [IOpipe Tracing Plugin Documentation](https://github.com/iopipe/iopipe-plugin-trace#readme)
 
+If you would like to override a plugin for a purpose such as testing out a beta release, you can install the plugin directly and it will be picked up automatically, like:
+`npm install @iopipe/trace@autoHttp`
```

Comment:
Should we generalize this to something we don't have a release channel for? `@iopipe/trace@TAG`